opendeplete/openmc_wrapper.py

```diff
@@ -859,7 +713,8 @@ def unpack_tallies_and_normalize(self, filename):
             slab = materials.index(mat)
 
             # Get material results hyperslab
-            results = file["tallies/tally 1/results"][slab, :, 0]
+            arr = openmc.capi.tallies[1].results
+            results = arr[slab, :, 1]
```

Comment:
I know I shouldn't use magic numbers, but why did it shift from ``[slab, :, 0]`` to ``[slab, :, 1]``?  Is this just Fortran/C indexing stuff?

Reply:
This has to do with how tally results were stored on disk and how they are stored in memory. When OpenMC is running, the tally results array keeps three entries for each tally score:

1. a temporary value that is accumulated during a single batch
2. the sum of batch values
3. the sum of squared batch values

When the results are written to disk, only 2 and 3 get written, so the 'sum' value is effectively stored in index 0. However, in memory, the temporary value is in index zero so the 'sum' value is index 1.